committee-code-of-conduct/election.md

```diff
@@ -28,10 +28,9 @@ This document outlines the process, for Code of Conduct Committee (CoCC) electio
 The steering committee will announce a request for nominations some time in July from anyone meeting the [eligibility for candidacy](#eligibility-for-candidacy) guidelines.
 
 The steering committee will accept nominations for at least 1 week. Though the steering
-committee may choose to accept nominations for longer.  After the announced nomination
-period the steering committee will hold a private ballot election using time-limited
-[Condorcet] ranking use the [Schulze method] to break ties. The steering committee
-will publicly announce the results of the election.
+committee may choose to accept nominations for longer. Once the announced nomination
+period ends, steering committee will hold an election using [Elekto](https://elections.k8s.io/)
```

Comment:
I wonder if we should dedupe this to more generally document our current elections implementation, but that's not important at the moment.

Reply:
I agree we can have a relook at the election doc to de-dup content at some places. We can take it as a follow-up.